### PR TITLE
Stub out copilotRelated command

### PR DESCRIPTION
**Pull request by sandersn**
This feature didn't help, but it took a long time to verify that.
The copilotRelated command will return an empty array in 5.7, and should be removed entirely in Typescript 5.8.

The only client should be VSCode 1.95. Matching PR to remove it from the typescript extension: https://github.com/microsoft/vscode/pull/233729.

### CODE REVIEW

File: tests/baselines/reference/api/typescript.d.ts [hunk 1]
```diff
@@ -123,6 +123,7 @@ declare namespace ts {
                 ProvideInlayHints = "provideInlayHints",
                 WatchChange = "watchChange",
                 MapCode = "mapCode",
+                /** @deprecated */
```
Comment (jakebailey) [inline tests/baselines/reference/api/typescript.d.ts:126]: Maybe just make these internal in the API? This was never in a stable release so nobody should depend on compiling it.

Comment (jakebailey) [reply tests/baselines/reference/api/typescript.d.ts:126]: Even deleting the types below may be okay given they aren't used.

**Comment by sandersn** [comment]
@typescript-bot cherry-pick this

**Comment by sandersn** [comment]
@typescript-bot cherry-pick

**Comment by sandersn** [comment]
@typescript-bot cherry-pick to release-5.7

**Pull request merged**